src/components/Thanks/ThanksPageShare.vue

```diff
@@ -70,11 +77,25 @@
 					<template v-else>
 						<h1	class="thanks__headline-h1 tw-mt-1 tw-mb-3 tw-text-left">
 							<!-- eslint-disable-next-line max-len -->
-							Can you share this loan with one more person?
+							<template v-if="categoryShareVersion === 'a' && categoryName">
+								<!-- eslint-disable-next-line max-len -->
+								<span class="fs-mask">{{ this.lender.firstName }}</span>, share now to find allies in the fight against economic inequity for {{ this.categoryName }}.
+							</template>
+							<template v-else-if="categoryShareVersion === 'b' && categoryName">
+								More loans like yours mean more opportunities for {{ this.categoryName }}.
+							</template>
+							<template v-else>
+								Can you share this loan with one more person?
+							</template>
 						</h1>
 						<p class="tw-text-h3 tw-m-0 thanks__base-text">
-							<!-- eslint-disable-next-line max-len -->
-							<span class="fs-mask">{{ this.loan.name }}</span> only needs {{ calculatePeopleQtyToGoal() }} more people to lend $25 and their loan could be fully funded in a matter of hours!
+							<template v-if="categoryShareVersion === 'c' || !categoryName">
+								<!-- eslint-disable-next-line max-len -->
+								<span class="fs-mask">{{ this.loan.name }}</span> only needs {{ calculatePeopleQtyToGoal() }} more people to lend $25 and their loan could be fully funded in a matter of hours!
```

Comment:
Do we need to mask the loan name here?

Reply:
You're right, don't think it's necessary!